pkg/framework/operators/baseoperator.go

```diff
@@ -52,6 +52,7 @@ type BaseOperator struct {
 	cRoleBinding      rbacv1.ClusterRoleBinding
 	crds              []apiextv1b1.CustomResourceDefinition
 	keepCRD           bool
+	clientImageName	  string
```

Comment:
Same here. The baseoperator does not implicitly needs a client image. This has to be handled separately from the baseoperator.